_security-plugin/configuration/yaml.md

```diff
@@ -130,6 +131,12 @@ The opensearch.yml file also contains the `plugins.security.allow_default_init_s
 plugins.security.allow_default_init_securityindex: true
 ```
 
+Authentication cache for the security plugin exists to help speed up authentication by temporarily storing user credentials, thereby bypassing the authentication backend and the response times involved with that process. You can set a timeout value for cashing in minutes with the `plugins.security.cache.ttl_minutes` property. Default is `60`. You can disable caching by adding the value `0`.
```

Comment:
You can set a timeout value for *caching*

Reply:
@kolchfa-aws Thanks for catching.